src/main/java/screens/ListingsPopUp.java

```diff
@@ -0,0 +1,27 @@
+package screens;
+
+import useCases.displayListing.DisplayListingDsRequestModel;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.List;
+
+public class ListingsPopUp extends JDialog{
+
+    public ListingsPopUp(List<DisplayListingDsRequestModel> filteredListings) {
```

Comment:
Great idea to display the listings in a pop-up, makes a lot of sense and makes it easier to change!

Reply:
thank you!